src/software/pipeline/main_imageMatching.cpp

```diff
@@ -594,18 +594,17 @@ int main(int argc, char** argv)
           }
         }
 
-        if((matchingMode == EImageMatchingMode::A_A_AND_A_B) ||
+        if((matchingMode == EImageMatchingMode::A_AB) ||
            (matchingMode == EImageMatchingMode::A_B))
         {
-          nbFeaturesLoadedInputB = aliceVision::voctree::populateDatabase<DescriptorUChar>(sfmDataB, featuresFolders, tree, dbB, nbMaxDescriptors);
-          nbSetDescriptors +=  dbB.getSparseHistogramPerImage().size();
+          nbFeaturesLoadedInputB = aliceVision::voctree::populateDatabase<DescriptorUChar>(sfmDataB, featuresFolders, tree, db, nbMaxDescriptors);
+          nbSetDescriptors +=  db.getSparseHistogramPerImage().size();
```

Comment:
As we accumulate in the same DB size() will return the full number of images and not only the ones from B, isn't it?

Reply:
yes, I just corrected in the next commit.
I hadn't seen it, `nbSetDescriptors` is only used for printing.